code/game/turfs/flooring/flooring_premade.dm

```diff
@@ -177,13 +181,15 @@
 /turf/simulated/floor/airless/lava
 /turf/simulated/floor/light
 /turf/simulated/floor/snow
-/turf/simulated/floor/beach/coastline
```

Comment:
the coastline thing here was a duplicate definition, its just a couple lines down in the same file